apps/src/blockly/addons/contextMenu.js

```diff
@@ -209,10 +252,11 @@ const themes = [
     label: msg.blocklyTritanopiaTheme(),
   },
 ];
+const cursors = ['default', 'basic', 'line'];
```

Comment:
nit: Consider moving this to a Blockly-related constants file and making it all caps.
```
const CURSOR_TYPES = [...]
```